pkg/cli/admin/release/info.go

```diff
@@ -1987,6 +2035,298 @@ func describeBugs(out, errOut io.Writer, diff *ReleaseDiff, dir string, format s
 	return nil
 }
 
+func (o *InfoOptions) listRpmdb(releaseInfo *ReleaseInfo, dir, format, targetImage string) error {
+	if targetImage == "" {
+		// find image for machine-os component
+		if tag, ok := releaseInfo.ComponentTags["machine-os"]; !ok {
+			return fmt.Errorf("component machine-os not found in release")
+		} else {
+			targetImage = tag
+		}
+	}
+
+	var err error
+	var targetTag *imageapi.TagReference
+	for _, tag := range releaseInfo.References.Spec.Tags {
+		if tag.Name == targetImage {
+			targetTag = &tag
+			break
+		}
+	}
+	if targetTag == nil {
+		return fmt.Errorf("tag %s not found in release", targetImage)
+	}
+
+	ref, err := imagereference.Parse(targetTag.From.Name)
+	if err != nil {
+		return err
+	}
+
+	// not sure this can ever happen, but just sanity-check anyway
+	if ref.ID == "" {
+		return fmt.Errorf("image not digest-based: %s", ref)
+	}
+
+	dbpath := filepath.Join(dir, ref.ID)
+	if _, err = os.Stat(dbpath); os.IsNotExist(err) {
+		err = o.downloadRpmdb(ref.Name, dbpath)
```

Comment:
shouldn't this just use `targetTag.From.Name` instead? ref.Name only points to the image name without the repo or the tag. i get an `unable to connect to image repository : invalid reference format` when ref.Name is used

Reply:
Fixed, thanks! I hadn't retested a change I did in that area correctly because I was relying on cache.